typescript/libs/lsp_client.py

```diff
@@ -243,31 +293,37 @@ def __init__(self, binary_path, args, env, root_path):
         The script file to run is passed to the constructor.
         """
         super(ServerClient, self).__init__(binary_path, args, env, root_path)
-        print("Trying to spawn executable from: " + binary_path)
+        log.debug('Trying to spawn {0}'.format(' '.join(self.args)))
         try:
             if os.name == "nt":
                 # linux subprocess module does not have STARTUPINFO
                 # so only use it if on Windows
                 si = subprocess.STARTUPINFO()
                 si.dwFlags |= subprocess.SW_HIDE | subprocess.STARTF_USESHOWWINDOW
                 self.server_proc = subprocess.Popen(self.args,
-                                                    stdin=subprocess.PIPE, stdout=subprocess.PIPE, startupinfo=si, cwd=self.root_path, env=self.env)
+                                                    stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE, startupinfo=si, cwd=self.root_path, env=self.env)
             else:
                 log.debug("opening " + binary_path)
                 self.server_proc = subprocess.Popen(self.args,
-                                                    stdin=subprocess.PIPE, stdout=subprocess.PIPE, cwd=self.root_path, env=self.env)
+                                                    stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE, cwd=self.root_path, env=self.env)
         except Exception as err:
             print(err)
             self.server_proc = None
         # start reader thread
         if self.server_proc and (not self.server_proc.poll()):
-            log.debug("server proc " + str(self.server_proc))
+            log.debug("server process " + str(self.server_proc.pid))
             log.debug("starting reader thread")
             readerThread = threading.Thread(target=ServerClient.__reader, args=(
                 self.server_proc.stdout, self.msgq, self.eventq, self.asyncReq, self.reqType, self.server_proc, self.event_handlers))
             readerThread.daemon = True
             readerThread.start()
-        self.postCmd(lsp_helpers.init_message("file://" + self.root_path, self.server_proc.pid))
+            log.debug("starting logger thread")
```

Comment:
On my machine, this error logger froze Sublime until I commented it out. I can't figure out why - any thoughts?